### PR TITLE
Adding main styles for the app 

### PR DESCRIPTION
**Pull request by elasfarc**
Final Css touches.

- Improved the look and feel of the application, adding main styles for the app using plain CSS.
- Update the readme file so it includes complete step-by-step instructions about how to clone and run.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,75 @@
...
+The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+All data is preserved thanks to the external. [Leaderboard API service.](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3)
+
+  
+
+[live version](https://elasfarc.github.io/Leaderboard/dist/)
+
+  
+
+![screen](https://raw.githubusercontent.com/elasfarc/Leaderboard/styling/ss3.png)
```
Comment (MahmoudBakr23) [inline README.md:17]: - [x] **OPTIONAL**
Please update the live version to the latest updates of the app design. It looks awesome great job 😉

File: README.md [hunk 2]
```diff
@@ -1,2 +1,75 @@
...
+
+`git clone https://github.com/elasfarc/To-Do-list.git`
+
+  
+
+To install all the available dependecies run:
+
+`npm install`
+
+
```
Comment (MahmoudBakr23) [inline README.md:49]: - [x] Great job explaining how to run the app locally in such a friendly way. However, I'm afraid you might have missed the `npm run webpack` CLI for building the code files to the _dist_ directory. 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @elasfarc 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct
- Great design that follows the given wireframe

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

- [x] Kindly always keep in mind to not leave the PR headline with its default value (the branch name). Instead, kindly write a more descriptive headline for the current PR 🙏 
- [x] Please write a more descriptive PR summary covering your final touches to your app. I recommend mentioning the adjustments you did to the README file, and the awesome design you made to the app 🥰 

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZawadiSumba66 — APPROVED** [review]
### Hi @elasfarc 👋 , 
## STATUS: APPROVED 👍 
Great job implementing changes requested by the previous reviewer. Well-don!! Keep rocking 🚀 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**